tests/Api/Responses/Expected/shop/order/get_order_item_response.json

```
@@ -5,75 +5,7 @@
     "variant": "\/api\/v2\/shop\/product-variants\/MUG_BLUE",
     "productName": "Mug",
     "id": @integer@,
-    "order": {
-        "@id": "\/api\/v2\/shop\/orders\/token",
-        "@type": "Order",
-        "shippingAddress": {
-            "@id": "\/api\/v2\/shop\/addresses\/@integer@",
-            "@type": "Address",
-            "firstName": "John",
-            "lastName": "Doe",
-            "phoneNumber": null,
-            "company": null,
-            "countryCode": "US",
-            "provinceCode": null,
-            "provinceName": null,
-            "street": "Avenue",
-            "city": "New York",
-            "postcode": "90000"
-        },
-        "billingAddress": {
-            "@id": "\/api\/v2\/shop\/addresses\/@integer@",
-            "@type": "Address",
-            "firstName": "John",
-            "lastName": "Doe",
-            "phoneNumber": null,
-            "company": null,
-            "countryCode": "US",
-            "provinceCode": null,
-            "provinceName": null,
-            "street": "Avenue",
-            "city": "New York",
-            "postcode": "90000"
-        },
-        "payments": [
-            {
-                "@id": "\/api\/v2\/shop\/payments\/@integer@",
-                "@type": "Payment",
-                "id": @integer@,
-                "method": "\/api\/v2\/shop\/payment-methods\/CASH_ON_DELIVERY"
-            }
-        ],
-        "shipments": [
-            {
-                "@id": "\/api\/v2\/shop\/shipments\/@integer@",
-                "@type": "Shipment",
-                "id": @integer@,
-                "method": "\/api\/v2\/shop\/shipping-methods\/UPS"
-            }
-        ],
-        "currencyCode": "USD",
-        "localeCode": "en_US",
-        "checkoutState": "completed",
-        "paymentState": "awaiting_payment",
-        "shippingState": "ready",
-        "tokenValue": "token",
-        "id": @integer@,
-        "number": "000000001",
-        "items": [
-            "\/api\/v2\/shop\/order-items\/@integer@"
-        ],
-        "itemsTotal": 6000,
-        "total": 6500,
-        "itemsSubtotal": 6000,
-        "taxTotal": 0,
-        "shippingTaxTotal": 500,
-        "taxExcludedTotal": 0,
-        "taxIncludedTotal": 0,
-        "shippingTotal": 500,
-        "orderPromotionTotal": 0,
-        "shippingPromotionTotal": 0
-    },
+    "order": "\/api\/v2\/shop\/orders\/token",
```

```suggestion
    "order": "\/api\/v2\/shop\/orders\/@string@",
```
unless the token is asserted anywhere.
A bit of a different problem here as well;
Why is the shop's item get on `shop/order-items` while changing quantity and removing is on `shop/order/{token}/items/{id}`?